keyboards/splitkb/kyria/keymaps/mnil/keymap.c

```diff
@@ -0,0 +1,117 @@
+        _______, M_TILD,  M_CIRC,  SE_LABK, SE_RABK, KC_NO  , KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS, SE_QUOT, SE_SLSH, SE_BSLS, SE_ASTR, M_BTCK,  _______,
+                                   _______, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, _______
+    ),
+    [_NAVIGATION] = LAYOUT(
+        _______, KC_NO,   CUT,     COPY,    PASTE,   AUTOFILL,                                        KC_WH_L, KC_HOME, KC_WH_D, KC_WH_U, KC_END,  _______,
+        _______, KC_ACL0, KC_ACL1, KC_BTN2, KC_BTN1, KC_LCTL,                                         KC_WH_R, KC_MS_L, KC_MS_D, KC_MS_U, KC_MS_R, _______,
+        _______, KC_ACL2, KC_BTN4, KC_BTN3, KC_BTN5, KC_NO,   KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS, KC_NO,   KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT, _______,
+                                   _______, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, _______
+    ),
+    [_NUMPAD] = LAYOUT(
+        _______, QUIT,    WIN,     MVWSL  , MVWSR,   CRYWS,                                           TERM,    KC_7,    KC_8,    KC_9,    KC_COMM, _______,
+        _______, CS_TAB,  C_TAB,   PRVWS,   NXTWS,   I3MOD,                                           OPEN,    KC_4,    KC_5,    KC_6,    KC_0,    _______,
+        _______, KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT, MOVWS,   KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS, BROWSER, KC_1,    KC_2,    KC_3,    KC_DOT,  _______,
+                                   _______, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, _______
+    )
+    // clang-format on
+};
+
+bool encoder_update_user(uint8_t index, bool clockwise) {
+    if (index == 0) {
+        if (clockwise) {
+            tap_code(KC_WH_U);
+        } else {
+            tap_code(KC_WH_D);
+        }
+    } else if (index == 1) {
+        if (clockwise) {
+            tap_code(KC_WH_R);
+        } else {
+            tap_code(KC_WH_L);
+        }
+    }
+    return false;
+}
+
+bool oled_task_user(void) {
+    if (is_keyboard_master()) {
+        // clang-format off
+        static const char PROGMEM qmk_logo[] = {
+            0x80,0x81,0x82,0x83,0x84,0x85,0x86,0x87,0x88,0x89,0x8a,0x8b,0x8c,0x8d,0x8e,0x8f,0x90,0x91,0x92,0x93,0x94,
+            0xa0,0xa1,0xa2,0xa3,0xa4,0xa5,0xa6,0xa7,0xa8,0xa9,0xaa,0xab,0xac,0xad,0xae,0xaf,0xb0,0xb1,0xb2,0xb3,0xb4,
+            0xc0,0xc1,0xc2,0xc3,0xc4,0xc5,0xc6,0xc7,0xc8,0xc9,0xca,0xcb,0xcc,0xcd,0xce,0xcf,0xd0,0xd1,0xd2,0xd3,0xd4,0};
+        // clang-format on
+
+        oled_write_P(qmk_logo, false);
+        oled_write_P(PSTR("Mats Nilsson\n\n"), false);
+
+        // Host Keyboard Layer Status
+        oled_write_P(PSTR("Layer: "), false);
+        switch (get_highest_layer(layer_state)) {
+            case _COLEMAK:
+                oled_write_P(PSTR("Colemak\n"), false);
+                break;
+            case _QWERTY:
+                oled_write_P(PSTR("Qwerty\n"), false);
+                break;
```

Comment:
Is this supposed to be here? I don't see a `_QWERTY` layer in your keymap.

Reply:
Given the "include mnil" at the top, sounds like it's in the userspace.